webapp/src/components/CreateMeetingNoteForm.tsx

```diff
@@ -0,0 +1,96 @@
+import { Alert, Stack, TextField, Typography } from '@mui/material';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import { LoadingButton } from '@mui/lab';
+import React, { FormEventHandler, useState } from 'react';
+
+import { EntityId } from '../types/api';
+interface CreateMeetingNoteFormProps {
+  agendaOwningParticipantId: EntityId;
+  meetingId: EntityId;
+  onMeetingNoteChanged?: (id: EntityId) => void;
+  sortOrder: number;
+}
+
+const CreateMeetingNoteForm = ({
+  agendaOwningParticipantId,
+  meetingId,
+  onMeetingNoteChanged,
+  sortOrder,
+}: CreateMeetingNoteFormProps) => {
+  const [isSavingMeetingNote, setIsSavingMeetingNote] = useState(false);
+  const [saveMeetingNoteError, setSaveMeetingNoteError] = useState(null);
+  const [meetingNoteText, setMeetingNoteText] = useState('');
+  const [isSuccessIndicated, setisSuccessIndicated] = useState(false);
+
+  const onSubmit: FormEventHandler = event => {
+    event.preventDefault();
+    setIsSavingMeetingNote(true);
+    fetch(`${process.env.REACT_APP_API_ORIGIN}/meetings/${meetingId}/notes`, {
+      method: 'POST',
+      credentials: 'include',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        agenda_owning_participant_id: agendaOwningParticipantId,
+        note_text: meetingNoteText,
+        sort_order: sortOrder,
+      }),
+    })
+      .then(res => res.json())
+      .then(({ data, error }) => {
+        setIsSavingMeetingNote(false);
+        if (error) {
+          setSaveMeetingNoteError(error);
+        } else {
+          setSaveMeetingNoteError(null);
+        }
```

Comment:
Would the following do pretty much the same thing?

```suggestion
        setSaveMeetingNoteError(error);
```

Reply:
I don't believe so... The suggestion always sets the `SaveMeetingNoteError`, which could be `undefined` when the `POST` is successful, or an object.  That said, I don't think lines 43-45 are necessary.

Reply:
Does it matter if `SaveMeetingNoteError` is set to `undefined` or `null`?

Reply:
How is it being used? Is its use dependent on it being exactly `null` or would `undefined` be no different?

Reply:
It's being used in the `catch` statement of the fetch call and to conditionally render the error message.  I don't think it matters either way.  To make the code a little more concise, I will initialize the variable to `undefined` in order to have only two conditions for this variable.